examples/s/d/nn/xcase/realization-0/iter-0/rms/bin/export_faultroom_surfaces.py

```diff
@@ -0,0 +1,43 @@
+"""Export faultroom surfaces via dataio with metadata."""
+
+import logging
+from pathlib import Path
+
+import fmu.dataio as dataio
+from fmu.config import utilities as utils
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.WARNING)
+
+CFG = utils.yaml_load("../../fmuconfig/output/global_variables.yml")
+
+HORISONNAMES = CFG["rms"]["horizons"]["TOP_RES"]
+
+# if inside RMS
+RMS_POL_CATEGORY = "GL_faultlines_extract_postprocess"
+
+# if running outside RMS using files that are stored e.g. on rms/output
+FAULTROOM_FILE = Path("../output/faultroom/some_faultroom.json")
+
+
+def export_faultroom_surface():
+    """Export faultroom data, json files made by FaultRoom plugin in RMS"""
+
+    ed = dataio.ExportData(
+        config=CFG,
+        content="fault_properties",
+        unit="unset",
+        vertical_domain={"depth": "msl"},
+        is_prediction=True,
+        is_observation=False,
+        workflow="rms structural model",
+    )
+
+    # read file and return a FaultRoomSurface instance
+    faultroom_object = dataio.readers.read_faultroom_file(FAULTROOM_FILE)
```

Comment:
This is so nice, love it!

Reply:
yes, agree - and this is also very consistent with the established pattern of 1) representing whatever data as a Python object, then 2) feeding that Python object to `.export()`.